### ISSUE TITLE
Plan out integration with CC Vocabulary

### ISSUE THREAD

**Issue by kgodey** [creativecommons/wp-plugin-creativecommons#85]
We should use CC Vocabulary for all the UI components offered by this plugin. This issue is to write up a spec and create the associated GitHub issues for what that would entail.

**Comment by Rajesh-Royal** [comment]
I would like to work on this feature.

**Comment by kgodey** [comment]
Go ahead and write a spec for it, @Rajesh-Royal

**Comment by Rajesh-Royal** [comment]
Thanks, I will write it when I finished Implementing CC-image-search in plugin by cc-catalogue API

**Comment by Rajesh-Royal** [comment]
can we use [lenasterg](https://github.com/lenasterg/ls-wp-ccsearch) codebase to kickstart the process.

**Comment by kgodey** [comment]
@Rajesh-Royal that repo isn't related to Vocabulary

**Comment by Rajesh-Royal** [comment]
> @Rajesh-Royal that repo isn't related to Vocabulary

No, I meant to add CC-image-search feature to the plugin by catalogue API. Lensaterg already have a sample codebase.

**Comment by kgodey** [comment]
@Rajesh-Royal please discuss that on the appropriate issue, this is unrelated.

**Comment by possumbilities** [comment]
Closing due to imminent archiving
